woocommerce/woocommerce-ios
# [Woo Express Sign up] Stop presenting upgrades flow from expired alert

Closes: #12392 


## Description

Stop presenting plan purchase or upgrade flow for an expired WPCOM plan site. 

Internal discussion - p1712052066222499/1712027364.219649-slack-C03L1NF1EA3

## Testing instructions

Prerequisites
- You need an expired WooExpress store connected with your WPCOM account

Steps
- Login into the app using a store which is not expired
- Navigate to "Menu" 
- Tap on store name to open store picker
- Try switching to an expired WooExpress store 
- Validate that you see an alert asking to purchase plan from WordPress.com
- Notice that the alert only has "OK" button and doesn't launch any actionable purchase plan flow

## Screenshots


<img src="https://github.com/woocommerce/woocommerce-ios/assets/524475/c40f9918-de35-4f79-8542-fc6101b31418" width="320"/>

---
